### DISCUSSION TITLE
playground broken?

### DISCUSSION THREAD

**Discussion by anon-ghub** [prisma/prisma#25715] [Q&A]
### Question

I can't use the playground. I used it a few days ago, but every since, when I get to the start of tutorial, a blank page shows with a header.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by anon-ghub** [comment]
Also, when I try to visit the slack link, which redirects to Discord, and then it isn't possible to actually join the discord.

**Reply by ludralph** [reply]
Hi @ka-azimi-tech 

I have raised the issue of Prisma Playground not working internally. I'll provide additional updates once I have one. Meanwhile, you can join our discord using this link https://pris.ly/discord

**Comment by anon-ghub** [comment]
invite still doesn't work

**Reply by ludralph** [reply]
Hi @ka-azimi-tech 

Did you get any error message? Can you try this link https://discord.com/invite/prisma-937751382725886062

**Reply by anon-ghub** [reply]
this one also. it just says "whoops" with no detail. It's not important. I might use the discord once very few months.

**Comment by anon-ghub** [comment]
Thank you to whoever fixed this! It's much faster working through training exercises with the playground rather than creating new node projects locally.
